metadoc-js/src/main/scala/monaco/Monaco.scala

```diff
@@ -1638,131 +1316,223 @@ package editor {
+    var horizontalHasArrows: Boolean = js.native
+    var handleMouseWheel: Boolean = js.native
+    var horizontalScrollbarSize: Double = js.native
+    var verticalScrollbarSize: Double = js.native
+    var verticalSliderSize: Double = js.native
+    var horizontalSliderSize: Double = js.native
+  }
+
+  @js.native
+  trait IEditorFindOptions extends js.Object {
+    var seedSearchStringFromSelection: Boolean = js.native
+    var autoFindInSelection: Boolean = js.native
+  }
+
+  @js.native
+  trait IEditorMinimapOptions extends js.Object {
+    var enabled: Boolean = js.native
+    var showSlider: String = js.native
+    var renderCharacters: Boolean = js.native
+    var maxColumn: Double = js.native
+  }
+
+  @js.native
+  trait IEditorOptions extends js.Object {
+    var ariaLabel: String = js.native
+    var rulers: js.Array[Double] = js.native
+    var wordSeparators: String = js.native
+    var selectionClipboard: Boolean = js.native
+    var lineNumbers: String | js.Function1[Double, String] = js.native
+    var selectOnLineNumbers: Boolean = js.native
+    var lineNumbersMinChars: Double = js.native
+    var glyphMargin: Boolean = js.native
+    var lineDecorationsWidth: Double | String = js.native
+    var revealHorizontalRightPadding: Double = js.native
+    var roundedSelection: Boolean = js.native
+    var extraEditorClassName: String = js.native
+    var readOnly: Boolean = js.native
+    var scrollbar: IEditorScrollbarOptions = js.native
+    var minimap: IEditorMinimapOptions = js.native
+    var find: IEditorFindOptions = js.native
+    var fixedOverflowWidgets: Boolean = js.native
+    var overviewRulerLanes: Double = js.native
+    var overviewRulerBorder: Boolean = js.native
+    var cursorBlinking: String = js.native
+    var mouseWheelZoom: Boolean = js.native
+    var cursorStyle: String = js.native
+    var fontLigatures: Boolean = js.native
+    var disableLayerHinting: Boolean = js.native
+    var disableMonospaceOptimizations: Boolean = js.native
+    var hideCursorInOverviewRuler: Boolean = js.native
+    var scrollBeyondLastLine: Boolean = js.native
+    var automaticLayout: Boolean = js.native
+    var wordWrap: String = js.native
+    var wordWrapColumn: Double = js.native
+    var wordWrapMinified: Boolean = js.native
+    var wrappingIndent: String = js.native
+    var wordWrapBreakBeforeCharacters: String = js.native
+    var wordWrapBreakAfterCharacters: String = js.native
+    var wordWrapBreakObtrusiveCharacters: String = js.native
+    var stopRenderingLineAfter: Double = js.native
+    var hover: Boolean = js.native
+    var links: Boolean = js.native
+    var contextmenu: Boolean = js.native
+    var mouseWheelScrollSensitivity: Double = js.native
+    var multiCursorModifier: String = js.native
+    var accessibilitySupport: String = js.native
+    var quickSuggestions: Boolean | js.Any = js.native
+    var quickSuggestionsDelay: Double = js.native
+    var parameterHints: Boolean = js.native
```

Comment:
I think it is related with the [SignatureHelpProvider](https://microsoft.github.io/monaco-editor/api/interfaces/monaco.languages.signaturehelpprovider.html)

> The signature help provider interface defines the contract between extensions and the parameter hints-feature.